content/en/docs/quickstart.md

```diff
@@ -43,86 +43,96 @@ notation list --plain-http $IMAGE
+
+Create a json file named `trustpolicy.json` with the following content:
+
+```json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:wabbit-networks.io" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
 ```
 
-Confirm your container image fails validation. For example:
-
-```output
-$ notation verify --plain-http $IMAGE
-Error: trust certificate not specified
-2022/09/30 15:24:14 trust certificate not specified
-```
+For a Linux user, store file `trustpolicy.json` under directory `$HOME/.config/notation/`.
 
-This verification fails because you have not trusted the public key for your container image.
+For a Mac user, store file `trustpolicy.json` under directory `$HOME/Library/Application Support/notation/`.
 
-## Add a trusted public key
+For a Window user, store file `trustpolicy.json` under directory `C:\Users\<username>\AppData\Roaming\notation\`.
 
-Add the public from the certificate you generated earlier as a trusted certificate.
+Once the trust policy file is executed, it creates one trust policy named `wabbit-networks-images`. The trust policy with `registryScopes` set to `*` applies to all the artifacts in any registries. The level of signatureVerification is `strict`, which enforces a full validation. Any validation failure will fail the whole signature verification process. The trust store used for this policy is named `wabbit-networks.io` of type `ca`, which is created in previous step. See [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy-properties) to fine tune the policies for specific security requirements.
 
-```console
-notation cert add --name "wabbit-networks.io" <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
-```
+## Verify the container image
 
-Adding this public key as a trusted public key allows you to verify any container images that are signed with the corresponding private key. 
+Use `notification verify` to verify any signatures on your container image.
 
-```output
-$ notation verify --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
+```console
+notation verify --plain-http $IMAGE
 ```
 
-The above example shows one trusted signature.
-
 ## Next steps
 
 For additional examples on using Notary, see:
 
-- [Build, sign, and verify container images using Notary and Azure Key Vault](https://learn.microsoft.com/azure/container-registry/container-registry-tutorial-sign-build-push)
\ No newline at end of file
+* [Build, sign, and verify container images using Notary and Azure Key Vault](https://learn.microsoft.com/azure/container-registry/container-registry-tutorial-sign-build-push)
```

Comment:
Is this Ratify guide working with Notation 0.12.0-beta.1?

Reply:
> Is this Ratify guide working with Notation 0.12.0-beta.1?

Maybe we should delete this link. This is not added by this PR.